### ISSUE TITLE
Can't install from repository, cant build from source.

### ISSUE THREAD

**Issue by jdoig** [ywelsch/duckdb-prql#14]
installing via the repository doesn't work as it looks like the `/v0.9.2/linux_amd64_gcc4` doesn't exist in the repository.
```
D SET custom_extension_repository='http://welsch.lu/duckdb/prql/latest';
D install prql;
Error: HTTP Error: Failed to download extension "prql" at URL "http://welsch.lu/duckdb/prql/latest/v0.9.2/linux_amd64_gcc4/prql.duckdb_extension.gz"

Candidate extensions: "parquet", "sqlite", "sqlite3", "sqlite_scanner"
```

Copying the file from the remote repository to `/home/me/.duckdb/extensions/v0.9.2/linux_amd64_gcc4/` then loading it cuases a segmentation fault.

Building it locally and running it with the version of duck the build process makes results in this result when issuing the PRQL statement from the README.
```
free(): invalid next size (fast)
Aborted (core dumped)
``

**Comment by ywelsch** [comment]
Hi @jdoig, the `_gcc4` version is indeed not being built at the moment. I had some trouble getting the build for it to work (see https://github.com/ywelsch/duckdb-prql/blob/c0e847ab1010ec49f989650ccec98549e40a7fbf/.github/workflows/Linux.yml#L20).

Regarding the segfault, see https://github.com/ywelsch/duckdb-prql/issues/10

The statement from the README is indeed broken (but other smaller examples work), and I'm looking into it (Haven't figured out the problem yet, it happens with certain queries, but not others).

**Comment by ywelsch** [comment]
I've pushed a fix for the gcc4 build (https://github.com/ywelsch/duckdb-prql/commit/ee7a75936ff07fa938077b0aee649c8aa3fc2276), so in 30mins from now the linux_amd64_gcc4 extension should be available for duckdb 0.10.0 (I'm only doing releases on the latest duckdb).

For the segfault, follow #10.
